src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -27,6 +29,15 @@ static constexpr unsigned int MAX_CLICK_COUNT = 3;
 
 namespace winrt::Microsoft::Terminal::Control::implementation
 {
+    static constexpr TerminalInput::MouseButtonState toInternalMouseState(const Control::MouseButtonState& state)
+    {
+        return TerminalInput::MouseButtonState{

```

Comment:
using a struct for this was weird in the first place. glad we're shoving it further away :)